### PR TITLE
MM-14452 - When a non-sysadmin tries to navigate to a system console URL, the login screen flashes briefly before redirecting to Town Square

### PR DESCRIPTION
**Pull request by reflog**
#### Summary
proper redirect from admin console

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-14452

### CODE REVIEW

File: components/admin_console/__snapshots__/admin_console.test.jsx.snap [hunk 1]
```diff
@@ -214,6 +214,6 @@ exports[`components/AdminConsole should generate the routes 1`] = `
 
 exports[`components/AdminConsole should redirect to / when not system admin 1`] = `
```
Comment (streamer45) [inline components/admin_console/__snapshots__/admin_console.test.jsx.snap]: I guess redirecting to `/` is not the case anymore.

File: components/admin_console/admin_console.test.jsx [hunk 2]
```diff
@@ -45,6 +45,7 @@ describe('components/AdminConsole', () => {
     test('should redirect to / when not system admin', () => {
```
Comment (streamer45) [inline components/admin_console/admin_console.test.jsx]: Same here.

File: components/admin_console/__snapshots__/admin_console.test.jsx.snap (hunk 1 above)
Comment (reflog) [reply components/admin_console/__snapshots__/admin_console.test.jsx.snap]: 👍

File: components/admin_console/admin_console.test.jsx (hunk 2 above)
Comment (reflog) [reply components/admin_console/admin_console.test.jsx]: 👍

**Pull request merged**